src/region_labelling.rs

```diff
@@ -244,13 +244,18 @@ where
 
 #[cfg(test)]
 mod tests {
+    extern crate wasm_bindgen_test;
+
     use super::connected_components;
     use super::Connectivity::{Eight, Four};
     use crate::definitions::{HasBlack, HasWhite};
     use ::test;
     use image::{GrayImage, ImageBuffer, Luma};
+    #[cfg(target_arch = "wasm32")]
+    use wasm_bindgen_test::*;
 
-    #[test]
+    #[cfg_attr(not(target_arch = "wasm32"), test)]
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
     fn test_connected_components_eight_white_background() {
```

Comment:
This is a lot of work to annotate three tests. Does wasm-bindgen-test require annotating every single test you want to run this way, and are you expecting to increase the set of tests that gets run this way over time?

Reply:
Yes, agreed; this is quite verbose. And yes, wasm-bindgen-test needs this. However, ideally it wouldn't, so I'm going to do a bit more research to see if it is definitely needed and/or if there is something less verbose.

Re: "expecting to increase the set of tests that gets run this way over time", it would make sense to, so that we have full wasm coverage. Separate to this, I've started working on full wasm coverage, see: https://github.com/mikemoraned/imageproc/pull/3 . This has shown up another couple of bugs, I think related to OsRng usage. However, I'll open a separate PR for anything that comes from this.

Reply:
Ok, thanks for the information. I'm fine to merge this as-is.